.chloggen/transform-flatten-logs.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: processor/transform
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add `transform.flatten.logs` featuregate to give each log record a distinct resource and scope.
```

Comment:
Should we add a note here as well that the config option is also required to enable this? It may be clear enough from the README, so just a suggestion.

Reply:
I think a note about the config option would also be good in the subtext, just so users hopefully have enough info to use the feature from the changelog entry.